pipe/pipe.py

```diff
@@ -3,47 +3,61 @@
 import os
 import subprocess
 import uuid
+import warnings
 from bitbucket import Bitbucket
 from bitbucket_pipes_toolkit import Pipe, get_logger
 
 OWASP_PATH="/usr/share/dependency-check/bin/dependency-check.sh"
 
 logger = get_logger()
 schema = {
-    'SCAN_PATH': {'type': 'string', 'required': True},
-    'CVSS_FAIL_LEVEL': {'type': 'string', 'required': False},
-    'SUPPRESSION_FILE_PATH': {'type': 'string', 'required': False},
-    'OUTPUT_PATH': {'type': 'string', 'required': False},
-    'OSSINDEX_USERNAME': {'type': 'string', 'required': False},
-    'OSSINDEX_PASSWORD': {'type': 'string', 'required': False},
-    'DISABLE_OSSINDEX': {'type': 'boolean', 'required': False},
+    'SCAN_PATH': {'type': 'string', 'required': True, 'default': '.'},
+    'CVSS_FAIL_LEVEL': {'type': 'string', 'required': False, 'default': '1'},
+    'SUPPRESSION_FILE_PATH': {'type': 'string', 'required': False, 'default': './suppression.xml'},
+    'OUTPUT_PATH': {'type': 'string', 'required': False, 'default': './test-results/'},
+    'OSSINDEX_USERNAME': {'type': 'string', 'required': False, 'default': ''},
+    'OSSINDEX_PASSWORD': {'type': 'string', 'required': False, 'default': ''},
+    'DISABLE_OSSINDEX': {'type': 'boolean', 'required': False, 'default': False},
+    'EXTRA_ARGS': {'type': 'string', 'required': False, 'default': ''},
     }
 
 class OWASPDependencyCheck(Pipe):
-    owasp_failure = False;
+    owasp_failure   = False;
```

Comment:
I'm not super familiar with python dev tools but is there some sort of linter or prettier equivalent to pick things like this up?

Reply:
I haven't used such a tool. But wanted to keep the `=` aligned.
![image](https://github.com/aligent/owasp-dependency-check-pipe/assets/19620378/008b803f-603a-4d1f-9458-b225f0fa4337)

Reply:
Ah I see!